common/metrics/defs.go

```diff
@@ -1723,6 +1723,19 @@ const (
 	TaskBatchCompleteCounter
 	TaskProcessingLatency
 	TaskQueueLatency
+
+	TaskRequestsAllDomains
+	TaskLatencyAllDomains
+	TaskFailuresAllDomains
+	TaskDiscardedAllDomains
+	TaskAttemptTimerAllDomains
+	TaskStandbyRetryCounterAllDomains
+	TaskNotActiveCounterAllDomains
+	TaskLimitExceededCounterAllDomains
+	TaskProcessingLatencyAllDomains
+	TaskQueueLatencyAllDomains
+	TransferTaskMissingEventCounterAllDomains
```

Comment:
Can we follow convention in matching metrics names and rename them to `XXXPerDomain`?